equinox/_module.py

```diff
@@ -477,11 +477,32 @@ def __init__(self, *args, **kwargs):
                     if not has_abstract_name:
                         # Invariant: abstract classes have names beginning with
                         # `Abstract`.
-                        raise TypeError(
+                        main = (
                             "Abstract strict `eqx.Module`s must be named starting "
-                            f"with 'Abstract' or '_Abstract'. Got {name} when defining "
-                            f"{cls.__module__}.{cls.__qualname__}.",
+                            f"with 'Abstract' or '_Abstract'. Got {name} when defining"
+                            f"{cls.__module__}.{cls.__qualname__}."
```

Comment:
I think you're missing a space across these two lines? The class name will appear right up against the "defining".

Other than that, LGTM!

Reply:
fixed